config/notifiers.go

```diff
@@ -394,3 +401,45 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	}
 	return checkOverflow(c.XXX, "pushover config")
 }
+
+// StatusPageConfig configures notifications via StatusPage.
+type StatusPageConfig struct {
+	NotifierConfig `yaml:",inline"`
+
+	APIURL      string `yaml:"api_url"`
+	APIKey      Secret `yaml:"api_key"`
+	PageId      string `yaml:"page_id"`
+	ComponentId string `yaml:"component_id"`
+	Status      string `yaml:"status"`
```

Comment:
For clarification: We use alerts to update the component's status using the traffic lights approach of StatusPage (screenshot: https://www.statuspage.io/tour):

![downtime-components-91c52702d1c55f1b057c0d68d91f5536](https://cloud.githubusercontent.com/assets/248965/17325828/14b4cece-58ae-11e6-8269-4682128ed6ab.png)